src/test/java/com/wikia/webdriver/elements/mercury/components/discussions/common/category/CategoryPills.java

```diff
@@ -1,15 +1,20 @@
 package com.wikia.webdriver.elements.mercury.components.discussions.common.category;
 
 import com.wikia.webdriver.pageobjectsfactory.pageobject.BasePageObject;
+import lombok.Setter;
 import org.openqa.selenium.By;
 import org.openqa.selenium.WebElement;
 import org.openqa.selenium.support.FindBy;
 
 import javax.annotation.CheckForNull;
 import java.util.List;
+import java.util.Optional;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.Optional'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AUselessImportCheck)